cmd/cluster/core/dump.go

```diff
@@ -349,9 +350,10 @@ func DumpCluster(ctx context.Context, opts *DumpOptions) error {
 		&capiazure.AzureCluster{},
 		&capiazure.AzureMachine{},
 		&capiazure.AzureMachineTemplate{},
-		&capiopenstack.OpenStackCluster{},
-		&capiopenstack.OpenStackMachine{},
-		&capiopenstack.OpenStackMachineTemplate{},
+		&capiopenstackv1alpha1.OpenStackServer{},
```

Comment:
do we also need to add the openstackserver at https://github.com/openshift/hypershift/blob/main/cmd/install/assets/assets.go#L54-L58?

Reply:
done, thx